src/commands/CleanCommand.js

```diff
@@ -1,16 +1,22 @@
 import async from "async";
 import Command from "../Command";
 import FileSystemUtilities from "../FileSystemUtilities";
+import PackageUtilities from "../PackageUtilities";
 import PromptUtilities from "../PromptUtilities";
 import progressBar from "../progressBar";
 
 export default class CleanCommand extends Command {
   initialize(callback) {
+    this.packagesToClean = this.filteredPackages;
+    if (this.flags.includeFilteredDependencies) {
+      this.packagesToClean = PackageUtilities.addDependencies(this.packagesToClean, this.packageGraph);
```

Comment:
Yep, I mostly just blindly copied the pattern. I think if this were to be extended other commands in the future, it could make sense to pull the check up into Command.js, and modify the `filteredPackages` there, maybe? I haven't looked closely at that though.